### PR TITLE
Visible separator

### PR DESCRIPTION
**Pull request by trigg**
In answer to #211 

Allow for rendering a single thin line using primary color. Should be configurable from CSS and by default matchs system theme.

### CODE REVIEW

**Comment by ammen99** [comment]
Wouldn't it be better to make it so that we support both `spacingN` and `separatorN` as widgets, so that the user can select which one to use where?

File: src/panel/panel.cpp [hunk 1]
```diff
@@ -230,6 +231,20 @@ class WayfirePanel::impl
...
         }
+        std::string separator = "separator";
+        if (name.find(separator) == 0)
+        {
+            auto pixel_str = name.substr(separator.size());
+            int pixel = std::atoi(pixel_str.c_str());
+
+            if (pixel <= 0)
+            {
+                std::cerr << "Invalid separator, " << pixel << std::endl;
```
Comment (ammen99) [inline src/panel/panel.cpp:243]: Might be better to print `pixel_str` so that the user sees which string is problematic

**Review by ammen99 — APPROVED** [review]
LGTM, one small nit and we can merge.

File: src/panel/panel.cpp (hunk 1 above)
Comment (trigg) [reply src/panel/panel.cpp:243]: Shall I do the same to spacing while I'm here?

Comment (ammen99) [reply src/panel/panel.cpp]: That actually reminds me, can't we extract it into a function?

Comment (trigg) [reply src/panel/panel.cpp]: Is this implementation what you had in mind?

**Comment by mrsteve0924** [comment]
nice! cant wait to check it out.

**Review by ammen99 — APPROVED** [review]
Thanks!

**Pull request merged**